pnpm-lock.yaml

```diff
@@ -3975,6 +4045,51 @@ packages:
     resolution: {integrity: sha512-U9nH88a3fc/ekCF1l0/UP1IosiuIjyTh7hBvXVMHYgVcfGvt897Xguj2UOLDeI5BG2m7/uwyaLVT6fbtCwTyzw==}
     engines: {iojs: '>=1.0.0', node: '>=0.10.0'}
 
+  rolldown-vite@https://pkg.pr.new/vitejs/rolldown-vite@8638a59:
```

Comment:
I don't know why `rolldown-vite@https://pkg.pr.new/vitejs/rolldown-vite@8638a59` is being used here